lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -459,6 +460,15 @@ def non_fractional_currency?(currency)
       def emv_payment?(payment)
         payment.respond_to?(:emv?) && payment.emv?
       end
+
+      def error_code_from(response)
+        code = response['error']['code'] 
+        decline_code = response['error']['decline_code'] if code == 'card_declined'
```

Comment:
Although they are the same string in this case, technically they are in different domains. The string here is part of the Stripe API domain. `STANDARD_ERROR_CODE[:card_declined]` is in the standard error code domain. There is nothing stopping these values from being different, although convention will probably have them be the same value forever.

Up to you if you want me to use the standard one instead.